operation.go

```diff
@@ -330,7 +329,7 @@ func (o *Operation) Password(prompt string) ([]byte, error) {
 	o.t.EnterRawMode()
 	defer o.t.ExitRawMode()
 
-	b, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+	b, err := terminal.ReadPassword(int(o.cfg.Stdin.Fd()))
```

Comment:
It doesn't work and isn't likely to work. You're in raw mode for operations already. Why are you using the terminal library to read passwords? Just stop echoing characters for the line.